README.md

```diff
@@ -4,128 +4,336 @@ Daniel Ehrenberg, Igalia. Stage 3
 
 Thanks for help and feedback on this effort from Brendan Eich, Waldemar Horwat, Jaro Sevcik, Benedikt Meurer, Michael Saboff, Adam Klein and others.
 
-## Code samples:
-
-Find the nth prime:
-
-```js
-// Takes a BigInt as an argument and returns a BigInt
-function nthPrime(nth) {
-  function isPrime(p) {
-    for (let i = 2n; i < p; i++) {
-      if (p % i === 0n) return false;
-    }
-    return true;
-  }
-  for (let i = 2n; ; i++) {
-    if (isPrime(i)) {
-      if (--nth === 0n) return i;
-    }
-  }
-}
+## Contents
+1. [What Is It?](#what-is-it)
+2. [How Does It Work?](#how-does-it-work)
+	- [Syntax](#syntax)
+	- [Example: Calculating Primes](#example-calculating-primes)
+	- [Operators](#operators)
+	- [Comparisons](#comparisons)
+	- [Conditionals](#conditionals)
+	- [Other API Notes](#other-api-notes)
+3. [Gotchas & Exceptions](#gotchas--exceptions)
+	- [Interoperation with `Number` and `String`](#interoperation-with-number-and-string)
+	- [Rounding](#rounding)
+	- [Other Exceptions](#other-exceptions)
+4. [About the Proposal](#about-the-proposal)
+	- [Motivation, Or Why Do We Need Such Big Numbers?](#motivation-why-do-we-need-such-big-numbers)
+	- [Design Philosophy, Or Why Is This Like This?](#design-goals-or-why-is-this-like-this)
+	- [State of the Proposal](#state-of-the-proposal)
+
+
+## What Is It?
+
+`BigInt` is a new primitive that provides a way to represent whole numbers larger than 2^53, which is the largest number Javascript can reliably represent with the `Number` primitive.
+
+```js
+const x = Number.MAX_SAFE_INTEGER;
+// ↪ 9007199254740991, this is 1 less than 2^53
+
+const y = x + 1;
+// ↪ 9007199254740992, ok, checks out
+
+const z = x + 2
+// ↪ 9007199254740992, wait, that’s the same as above!
+```
+
+*_NOTE: This link doesn't go anywhere yet, but I envision it linking to a short document explaining 64-bit floats, probably using the illos from the JSConfEU talk._*
```

Comment:
That'd be great for this. Would it make sense to link to the slides and/or video to start?